src/app.js

```diff
@@ -1,21 +1,31 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
 
-module.exports = sum;
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+
+  if (!(normalizedURL.startsWith('/file'))) {
+    res.end('Make sure the file path starts with "/file/"!');
+  } else {
+    const fileName = normalizedURL.pathname.replace('/file', '')
+    || 'index.html';
```

Comment:
check this one more time. Is it work without error?
`/file/`
<img width="388" alt="Screenshot 2022-10-18 at 21 58 21" src="https://user-images.githubusercontent.com/47606379/196519884-6d682357-7996-4b4b-a647-339d0a69065d.png">